packages/nns/src/enums/governance.enums.ts

```diff
@@ -124,12 +124,17 @@ export enum NnsFunction {
   UpdateAllowedPrincipals = 35,
   RetireReplicaVersion = 36,
   InsertSnsWasmUpgradePathEntries = 37,
-  UpdateElectedReplicaVersions = 38,
+  ReviseElectedGuestosVersions = 38,
   BitcoinSetConfig = 39,
   UpdateElectedHostosVersions = 40,
   UpdateNodesHostosVersion = 41,
-  AddApiBoundaryNode = 43,
+  HardResetNnsRootToVersion = 42,
+  AddApiBoundaryNodes = 43,
   RemoveApiBoundaryNodes = 44,
+  /**
+   * @deprecated
+   */
+  UpdateApiBoundaryNodeDomain = 45,
```

Comment:
Where does this come from? (Or should I say where did it go?)

Reply:
Found it in the proto (the Rust files don't mention `= 45` at all, the protos at least mark them as `reserved`)

Reply:
As mentioned in the PR description, everything comes from https://github.com/dfinity/ic/blob/master/rs/nns/governance/proto/ic_nns_governance/pb/v1/governance.proto